### ISSUE TITLE
No support to AppImage format (can not automatically grab it) !

### ISSUE THREAD

**Issue by Nokia808** [jingyu9575/multithreaded-download-manager#83]
Hi. 
I would like, at 1st, to thank you too much for this extension ! Currently it is bugy, but very promising ! If fixing current bugs, finally you will create something very great !

I detect that your nice extension not able currently to cache AppImage automatically ! Any AppImage file when downloaded, it will go to built-in browser downloader & should be manually imported to extension !

Kindly, to fix this.

I'm on Fedora 30 Linux X 64 with Firefox.

**Comment by jingyu9575** [comment]
What is the URL of the AppImage file?

During my tests, some AppImage links works, while some can not be caught by the extension. The latter is because the server does not tell the file type. This is the same problem as #27, and there is still no way to fix it.

**Comment by Nokia808** [comment]
@jingyu9575 
Open following page:
https://www.libreoffice.org/download/appimage/
I used "Download Fresh" for "Standard version". Click on it.

**Comment by Nokia808** [comment]
@jingyu9575 
& here is bellow the URL:
https://libreoffice.soluzioniopen.com/stable/standard/LibreOffice-fresh.standard-x86_64.AppImage

**Comment by jingyu9575** [comment]
Yes this is one of the links that do not provide the file type. For the reason in #27, the extension cannot catch it.

**Comment by Nokia808** [comment]
@jingyu9575 
Hi. I wish the you are in good health.
Dear I detected change in "Turbo Download Manager" that seem to me related to this issue.
Last version of TDM yesterday published (version 0.5.4) with the following change log statement:

"Uses maginc bytes to determine file type even if the server is not providing"

See the following:
https://addons.mozilla.org/en-US/firefox/addon/turbo-download-manager/
https://add0n.com/turbo-download-manager-v2.html

Please examine it's code at:
https://github.com/inbasic/turbo-download-manager-v2/
Developer, now, updating his code on GitHub again .....
